WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -14,3 +14,14 @@
     end
   end
 end
+
+describe "Momentjs activation" do
+  let(:file_content) { File.read(Rails.root.join('app', 'assets', 'javascripts', 'application.js')) }
+  let(:locales) { I18n.available_locales - [:en] }
+
+  (I18n.available_locales - [:en]).each do |locale|
+    context "for " + locale.to_s do
+      it { expect(file_content).to include("//= require moment/#{locale}.js") }
```

Comment:
I'm fine with whatever that is really human-readable just in case someone experiencing that doesn't know that issue.